torch/fx/experimental/proxy_tensor.py

```diff
@@ -367,100 +422,32 @@ def __init__(self, tracer):
         self.sym_mode = ProxySymDispatchMode(tracer)
         self.trace_state = {}
 
-    def __torch_dispatch__(self, func_overload, types, args=(), kwargs=None):
+    def __torch_dispatch__(self, func, types, args=(), kwargs=None):
         with self.sym_mode.enable(False):
-            return self.inner_torch_dispatch(func_overload, types, args, kwargs)
+            return self.inner_torch_dispatch(func, types, args, kwargs)
 
     @contextmanager
     def restore(self):
         with self.sym_mode.enable(True):
             with super().restore():
                 yield
 
-    def inner_torch_dispatch(self, func_overload, types, args=(), kwargs=None):
+    def inner_torch_dispatch(self, func, types, args=(), kwargs=None):
         if not self.enable_tracing:
-            return func_overload(*args, **kwargs)
+            return func(*args, **kwargs)
 
-        if symbolic_shapes.is_symbolic_op(func_overload):
+        if symbolic_shapes.is_symbolic_op(func):
             with self.restore():
-                return symbolic_shapes.handle_symbolic_op(func_overload, args, kwargs)
+                return symbolic_shapes.handle_symbolic_op(func, args, kwargs)
 
-        func = func_overload.overloadpacket
         # We don't want to convert torch.tensor constants into tracing objects.
-        if func_overload == aten.lift.default:
+        if func == aten.lift.default:
             return args[0]
 
-        if func in [prim.device]:
-            return func_overload(*args, **kwargs)
+        if func in [prim.device.default]:
+            return func(*args, **kwargs)
```

Comment:
I've also forgotten why specifically only prim.device needs to be handled this way, and not any of the other accessor calls 🤔

Reply:
Honestly I don't even know why where we see these calls from lol (never looked into it).